breeze-complete

```diff
@@ -173,9 +173,6 @@ function breeze_complete::get_known_values_breeze() {
     -g | --github-repository)
         _breeze_known_values="${_breeze_default_github_repository}"
         ;;
-    -s | --github-image-id)
```

Comment:
Should `github-image-id` also be removed/replaced in `_breeze_long_options` as well?

Reply:
Naaa. That's legacy code. I would not worry too much - we are going to delete it shortly

Reply:
I hope :dagger: